R/mda_helpers.R

```diff
@@ -1,31 +1,51 @@
-#'@title create timeseries pointer
-#'@param variable a timeseries name (e.g., \code{wtr})
-#'@return a timseries string
-#'@export
-make_ts_variable <- function(variable){
-  ts_prefix <- get_ts_prefix()
-  ts_variable = paste0(ts_prefix, variable)
-  return(ts_variable)
+#' Translate a timeseries name from mda.streams to ScienceBase
+#' @param variable a timeseries name[s] in mda.streams lingo (e.g., \code{wtr})
+#' @return timseries name[s] in ScienceBase lingo (e.g., "\code{ts_wtr})
+make_ts_name <- function(variable){
```

Comment:
also adding checks to make_site_name, but more limited - checking the database part but not the site num, because that would require a [slow] ScienceBase query